src/components/Form/widgets/SelectWidget.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from 'react';
-import { Select } from '../../../..';
-import { FormWidgetProps } from '../../../unstable-typings';
+import { Select } from '../../..';
+import { FormWidgetProps } from '../';
 
 const SelectWidget = (props: FormWidgetProps) => {
 	const { id, options, value, disabled, readonly, onChange } = props;
 
 	const { enumOptions, enumDisabled } = options;
-	const selectOptions = enumOptions.map((option: any) => {
+	const selectOptions = (enumOptions as any[])?.map((option) => {
 		return {
 			...option,
-			disabled: enumDisabled && enumDisabled.indexOf(option.value) !== -1,
+			disabled:
+				enumDisabled && (enumDisabled as any[]).indexOf(option.value) !== -1,
```

Comment:
absolutely non-blocking, it's just a personal preference:
```suggestion
				(enumDisabled as any[])?.some(a => a === option.value),
```